wesnoth-thot.po

```diff
@@ -1,5 +1,5 @@
-# Vladimír Slávik <slavik.vladimir@seznam.cz>, 2007 - 2011.
-# Petr Šlejhar (Septim) <septim@centrum.cz>, 2008 - 2011.
+# Vladimír Slávik <slavik.vladimir@seznam.cz>, 2008 - 2011.
```

Comment:
Máte pro tenhle přepis historie nějakou oporu?

Já koukám do gitu, že [2008-02-10 přistálo do THoT](https://github.com/wesnoth/wesnoth/blob/b018d64b8dfc9f0505b5eb8a4347b671906d9fc1/po/wesnoth-thot/cs.po) prvních 52 českých zpráv a v hlavičce je uveden Vladimír 2007, 2008. Předpokládám, že to napsal on a může to znamenat, že s překladem začal před Novým rokem.

Reply:
THoT byla první kampaň, kterou jsem překládal. A mám utkvělý pocit, že od nuly, ale nechci zcela vylučovat, že tam pár řetězců už přeloženo bylo (ale 52?). Ostatně 12. 2. 2008 mi Vláďa odpovídal a chválil, jak je ten můj překlad skoro hotový. Tenkrát se ale všechno posílalo mailem Vláďovi (nebo výjimečně Otovi), takže tam nejsem uveden.
Asi tedy nemám problém nechat to tam Vláďovi od roku 2007 a také nevylučuji, že jsem kampaň začal i já překládat už na konci roku 2007.